spec/controllers/services_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe ServicesController, type: :controller do
+  let!(:ups) { ActiveShipping::UPS.new(login: ENV['UPS_LOGIN'], password: ENV['UPS_PASSWORD'], key: ENV['UPS_KEY']) }
+  let!(:usps) { ActiveShipping::USPS.new(login: ENV['ACTIVESHIPPING_USPS_LOGIN']) }
+  let(:keys)   { ["date", "price", "rate"] }
```

Comment:
You probably want to be more specific with your naming here. What do you mean keys? Keys for what?